rpm/gupnp.spec

```
@@ -38,32 +39,29 @@ Requires: pkgconfig
 Files for development with %{name}.
 
 %prep
-%setup -q -n %{name}-%{version}/%{name}
+%autosetup -p1 -n %{name}-%{version}/upstream
 
 %build
-%autogen --disable-static --with-context-manager=connman
-make %{?_smp_mflags} V=1
+%meson -Dcontext_manager=connman -Dexamples=false
+%meson_build
 
 %install
-make install DESTDIR=%{buildroot}
-
-#Remove libtool archives.
-find %{buildroot} -name '*.la' -exec rm -f {} ';'
+%meson_install
 
 %post -p /sbin/ldconfig
 
 %postun -p /sbin/ldconfig
 
 %files
 %doc AUTHORS COPYING README
```

Maybe having just license is enough:
```suggestion
%license COPYING
```